linkerd/failure-accrual/src/main/scala/io/buoyant/linkerd/failureAccrual/NoneInitializer.scala

```diff
@@ -0,0 +1,24 @@
+package io.buoyant.linkerd.failureAccrual
+
+import com.fasterxml.jackson.annotation.JsonIgnore
+import com.twitter.finagle.service.exp.FailureAccrualPolicy
+import com.twitter.util.Duration
+import io.buoyant.linkerd.{FailureAccrualConfig, FailureAccrualInitializer}
+
+class NoneInitializer extends FailureAccrualInitializer {
+  val configClass = classOf[NullConfig]
+  override def configId = "none"
+}
+
+object NoneInitializer extends NoneInitializer
+
+case class NullConfig() extends FailureAccrualConfig {
```

Comment:
Don't need to make this a case class, and you can call it NoneConfig for consistency.